src/pages/VideoCall/VideoCall.js

```diff
@@ -146,7 +142,23 @@ export const Controls = styled.div`
         flex-direction: column;
         justify-content: center;
         align-items: center;
-        margin-right: ${({ theme }) => theme.spacing.XXXL};
+        padding: 16px;
```

Comment:
personnally, I see the extra padding-bottom even without hovering. It disappears when replacing this padding value with `padding: 0 16px` 

It needs to be adapted for each breakpoint though, because after testing, 16px is too much padding on mobile for example